packages/lib/getEventTypeById.ts

```diff
@@ -37,7 +37,6 @@ export default async function getEventTypeById({
     name: true,
     username: true,
     id: true,
-    avatar: true,
```

Comment:
For team members this was patched already, but it still returned the avatar in the background, likely slowing down Prisma and data calls.